Gruntfile.js

```diff
@@ -13,6 +16,52 @@ grunt.initConfig({
 	},
 	jscs: {
 		all: [ "*.js" ]
+	},
+	sass: {
+		dist: {
+			options: {
+				style: "compressed"
+			},
+			files: [ {
+				expand: true,
+				cwd: "scss",
+				src: [ "*.scss" ],
+				dest: "",
+				ext: ".css"
+			} ]
+		}
+	},
+
+	autoprefixer: {
+		dist: {
+			options: {
+				map: true,
+
+				// Target-specific options go here.
+				// browser-specific info: https://github.com/ai/autoprefixer#browsers
+				// DEFAULT: browsers: [ "> 1%", "last 2 versions", "ff 17", "opera 12.1" ]
+				browsers: [
+				"> 1%",
+				"last 2 versions",
+				"ff 17",
+				"opera 12.1",
+				"ie 8",
+				"ie 9"
```

Comment:
This could change but as of right now our support is
IE8+
Chrome (Current - 1)
Firefox (Current - 1)
Opera (Current - 1)
Safari 5.1+
iOS 6.1+
Android 2.3, 4+

https://github.com/jquery/css-chassis/issues/3